apps/OboeTester/app/src/main/cpp/cpu/AudioWorkloadTest.h

```diff
@@ -168,7 +172,10 @@ class AudioWorkloadTest : oboe::AudioStreamDataCallback {
         mStream->setPerformanceHintEnabled(adpfEnabled);
         mStream->setBufferSizeInFrames(mNumBursts * mFramesPerBurst);
         mBufferSizeInFrames = mStream->getBufferSizeInFrames();
-        mSynthWorkload = SynthWorkload((int) 0.2 * mSampleRate, (int) 0.3 * mSampleRate);
+        {
+            std::lock_guard<std::mutex> synthWorkloadLock(mSynthWorkloadLock);
+            mSynthWorkload = std::make_unique<SynthWorkload>((int) 0.2 * mSampleRate, (int) 0.3 * mSampleRate);
+        }
         oboe::Result result = mStream->start();
         if (result != oboe::Result::OK) {
             std::cerr << "Error starting stream: " << oboe::convertToText(result) << std::endl;
```

Comment:
Need to set mRunning to false when there it fails to start

Reply:
Done! Thanks!